wagtail_streamfield_migration_toolkit/test/factories.py

```diff
@@ -0,0 +1,56 @@
+import factory
+import wagtail_factories
+
+try:
+    from factory.django import DjangoModelFactory
+except ImportError:
+    from factory import DjangoModelFactory
```

Comment:
Do you need this? This seems to be for factory_boy <3.0 - but 3.0 dropped support for Django 2.1 (which Wagtail no longer supports either) and adds support for 2.2 and 3.0 (which the versions of Wagtail you're working with need versions greater than at this point - see https://docs.wagtail.org/en/stable/releases/upgrading.html).

Reply:
right